src/backends/latex/print.jl

```diff
@@ -144,7 +145,11 @@ function _pt_latex(io::IO, pinfo::PrintInfo;
     # Print LaTeX header
     # ==========================================================================
 
-    if table_type == :tabular
+    if table_type != :tabular && wrap_table == true
```

Comment:
Wouldn't it be better to just ignore `wrap_table` if we are not using `:tabular`? In this version, the user will be forced to always use `table_type = :longtable, wrap_table = false`.

Reply:
I think it is good to throw an error, actually. 

I just checked in latex and 

```
\begin{table}
\begin{longtable}
... % loooong table
\end{longtable}
\end{table}
```

Doesn't work, as in the long-table doesn't extend to the next page. So allowing this just means that the user gets an error in latex which is really an error in this package.

Reply:
No, I mean, if the user selects `:longtable`, then never add the `table` environment. Something like this, if `table_type` is `:longtable`, then never wrap the table, even if `wrap_table` is `true`.

Reply:
It seems that you only need to delete this `if` that throws an error.

Reply:
oh. yeah definitely. Do you want a warning, even?

Reply:
No, it is not necessary. Just your comment that this option does not work with `tabular` is fine :)

Reply:
I plan in the future to add `tabularx`, `tabulary`, etc. In all cases, `warp_table` will work.  `longtable` seems to be the only exception here.